### PR TITLE
Fix csrf token on browser page

### LINKED ISSUE #697: [Branch 5.x] Ckeditor browse - csrf_token does not exist
### Environment

PHP 8.1.6
Symfony 5.4.6

#### Sonata packages

<details><summary>show</summary>
<p>

```
$ composer show --latest 'sonata-project/*'
sonata-project/admin-bundle              4.11.1          4.12.0          The missing Symfony Admin Generator
sonata-project/block-bundle              4.12.0          4.13.0          Symfony SonataBlockBundle
sonata-project/cache                     2.2.0           2.2.0           Cache library
Package sonata-project/cache is abandoned, you should avoid using it. No replacement was suggested.
sonata-project/doctrine-extensions       1.16.0          1.17.0          Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 4.2.8           4.3.0           Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  2.11.0          2.13.0          Lightweight Exporter library
sonata-project/form-extensions           1.13.1          1.17.1          Symfony form extensions
sonata-project/formatter-bundle          5.x-dev 42e5364 5.x-dev 8f58db8 Symfony SonataFormatterBundle
sonata-project/media-bundle              4.2.0           4.2.0           Symfony SonataMediaBundle
sonata-project/twig-extensions           1.9.1           1.10.0          Sonata twig extensions
```

</p>
</details>

#### Symfony packages

<details><summary>show</summary>
<p>

```
$ composer show --latest 'symfony/*'
symfony/asset                      v5.4.7  v6.1.0  Manages URL generation and versioning of web assets such as CSS stylesheets, JavaScript files ...
symfony/browser-kit                v5.4.3  v6.1.0  Simulates the behavior of a web browser, allowing you to make requests, click on links and sub...
symfony/cache                      v5.4.8  v6.1.1  Provides an extended PSR-6, PSR-16 (and tags) implementation
symfony/cache-contracts            v2.5.1  v3.1.0  Generic abstractions related to caching
symfony/config                     v5.4.8  v6.1.0  Helps you find, load, combine, autofill and validate configuration values of any kind
symfony/console                    v5.4.8  v6.1.1  Eases the creation of beautiful and testable command line interfaces
symfony/css-selector               v5.4.3  v6.1.0  Converts CSS selectors to XPath expressions
symfony/debug-bundle               v5.4.3  v6.1.0  Provides a tight integration of the Symfony VarDumper component and the ServerLogCommand from ...
symfony/dependency-injection       v5.4.8  v6.1.0  Allows you to standardize and centralize the way objects are constructed in your application
symfony/deprecation-contracts      v3.0.1  v3.1.0  A generic function and convention to trigger deprecation notices
symfony/doctrine-bridge            v5.4.8  v6.1.0  Provides integration for Doctrine with various Symfony components
symfony/dom-crawler                v5.4.6  v6.1.0  Eases DOM navigation for HTML and XML documents
symfony/dotenv                     v5.4.5  v6.1.0  Registers environment variables from a .env file
symfony/error-handler              v5.4.8  v6.1.0  Provides tools to manage errors and ease debugging PHP code
symfony/event-dispatcher           v5.4.3  v6.1.0  Provides tools that allow your application components to communicate with each other by dispat...
symfony/event-dispatcher-contracts v3.0.1  v3.1.0  Generic abstractions related to dispatching event
symfony/expression-language        v5.4.8  v6.1.0  Provides an engine that can compile and evaluate expressions
symfony/filesystem                 v5.4.7  v6.1.0  Provides basic utilities for the filesystem
symfony/finder                     v5.4.8  v6.1.0  Finds files and directories via an intuitive fluent interface
symfony/flex                       v2.1.8  v2.2.2  Composer plugin for Symfony
symfony/form                       v5.4.8  v6.1.1  Allows to easily create, process and reuse HTML forms
symfony/framework-bundle           v5.4.8  v6.1.1  Provides a tight integration between Symfony components and the Symfony full-stack framework
symfony/http-client                v5.4.8  v6.1.1  Provides powerful methods to fetch HTTP resources synchronously or asynchronously
symfony/http-client-contracts      v2.5.1  v3.1.0  Generic abstractions related to HTTP clients
symfony/http-foundation            v5.4.8  v6.1.1  Defines an object-oriented layer for the HTTP specification
symfony/http-kernel                v5.4.8  v6.1.1  Provides a structured process for converting a Request into a Response
symfony/intl                       v5.4.8  v6.1.0  Provides a PHP replacement layer for the C intl extension that includes additional data from t...
symfony/mailchimp-mailer           v5.4.7  v6.1.0  Symfony Mailchimp Mailer Bridge
symfony/mailer                     v5.4.8  v6.1.1  Helps sending emails
symfony/maker-bundle               v1.40.1 v1.43.0 Symfony Maker helps you create empty commands, controllers, form classes, tests and more so yo...
symfony/mime                       v5.4.8  v6.1.1  Allows manipulating MIME messages
symfony/monolog-bridge             v5.4.3  v6.1.1  Provides integration for Monolog with various Symfony components
symfony/monolog-bundle             v3.7.1  v3.8.0  Symfony MonologBundle
symfony/options-resolver           v5.4.3  v6.1.0  Provides an improved replacement for the array_replace PHP function
symfony/password-hasher            v5.4.8  v6.1.0  Provides password hashing utilities
symfony/phpunit-bridge             v6.0.8  v6.1.0  Provides utilities for PHPUnit, especially user deprecation notices management
symfony/polyfill-intl-grapheme     v1.25.0 v1.26.0 Symfony polyfill for intl's grapheme_* functions
symfony/polyfill-intl-icu          v1.25.0 v1.26.0 Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-intl-idn          v1.25.0 v1.26.0 Symfony polyfill for intl's idn_to_ascii and idn_to_utf8 functions
symfony/polyfill-intl-normalizer   v1.25.0 v1.26.0 Symfony polyfill for intl's Normalizer class and related functions
symfony/polyfill-mbstring          v1.25.0 v1.26.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php72             v1.25.0 v1.26.0 Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions
symfony/polyfill-php73             v1.25.0 v1.26.0 Symfony polyfill backporting some PHP 7.3+ features to lower PHP versions
symfony/polyfill-php80             v1.25.0 v1.26.0 Symfony polyfill backporting some PHP 8.0+ features to lower PHP versions
symfony/polyfill-php81             v1.25.0 v1.26.0 Symfony polyfill backporting some PHP 8.1+ features to lower PHP versions
symfony/process                    v5.4.8  v6.1.0  Executes commands in sub-processes
symfony/property-access            v5.4.8  v6.1.0  Provides functions to read and write from/to an object or array using a simple string notation
symfony/property-info              v5.4.7  v6.1.1  Extracts information about PHP class' properties using metadata of popular sources
symfony/proxy-manager-bridge       v5.4.6  v6.1.0  Provides integration for ProxyManager with various Symfony components
symfony/routing                    v5.4.8  v6.1.1  Maps an HTTP request to a set of configuration variables
symfony/runtime                    v5.4.8  v6.1.1  Enables decoupling PHP applications from global state
symfony/security-acl               v3.3.1  v3.3.1  Symfony Security Component - ACL (Access Control List)
symfony/security-bundle            v5.4.8  v6.1.0  Provides a tight integration of the Security component into the Symfony full-stack framework
symfony/security-core              v5.4.8  v6.1.0  Symfony Security Component - Core Library
symfony/security-csrf              v5.4.3  v6.1.0  Symfony Security Component - CSRF Library
symfony/security-guard             v5.4.3  v5.4.9  Symfony Security Component - Guard
symfony/security-http              v5.4.8  v6.1.1  Symfony Security Component - HTTP Integration
symfony/serializer                 v5.4.8  v6.1.1  Handles serializing and deserializing data structures, including object graphs, into array str...
symfony/service-contracts          v2.5.1  v3.1.0  Generic abstractions related to writing services
symfony/stopwatch                  v5.4.5  v6.1.0  Provides a way to profile code
symfony/string                     v5.4.8  v6.1.0  Provides an object-oriented API to strings and deals with bytes, UTF-8 code points and graphem...
symfony/templating                 v5.4.3  v6.1.0  Provides all the tools needed to build any kind of template system
symfony/translation                v5.4.8  v6.1.0  Provides tools to internationalize your application
symfony/translation-contracts      v2.5.1  v3.1.0  Generic abstractions related to translation
symfony/twig-bridge                v5.4.8  v6.1.0  Provides integration for Twig with various Symfony components
symfony/twig-bundle                v5.4.8  v6.1.1  Provides a tight integration of Twig into the Symfony full-stack framework
symfony/validator                  v5.4.8  v6.1.1  Provides tools to validate values
symfony/var-dumper                 v5.4.8  v6.1.0  Provides mechanisms for walking through any arbitrary PHP variable
symfony/var-exporter               v5.4.8  v6.1.1  Allows exporting any serializable PHP data structure to plain PHP code
symfony/web-link                   v5.4.3  v6.1.0  Manages links between resources
symfony/web-profiler-bundle        v5.4.8  v6.1.1  Provides a development tool that gives detailed information about the execution of any request
symfony/yaml                       v5.4.3  v6.1.0  Loads and dumps YAML files
```

</p>
</details>

#### PHP version

```
$ php -v
PHP 8.1.6 (cli) (built: May 12 2022 02:33:42) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.6, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.6, Copyright (c), by Zend Technologies
```

## Subject

The Ckeditor browser used to display the updated medias seems broken on the latest 5.x branch.
It shows a csrf_token missing error.

## Minimal repository with the bug

SonataFormatterBundle - commit 42e5364ab1465e65ea12f4b7fc1c8e84ea296cab
https://github.com/sonata-project/SonataFormatterBundle/tree/42e5364ab1465e65ea12f4b7fc1c8e84ea296cab

## Steps to reproduce

Install the Sonata stack with the version I use
Install Ckeditor
Configure a Sonata Admin with a Ckeditor field
Click the image button of Ckeditor
Click Browse to select an uploaded image

## Expected results

Display the list of uploaded medias

## Actual results

Error 500 - Variable "csrf_token" does not exist.
Screenshot 2022-06-17 at 19 05 06

## How to fix

Add a csrf_token property in the parameters return by browserAction of CkeditorAdminController
```
return $this->renderWithExtraParams($this->getTemplate('browser'), [
    'action' => 'browser',
    'form' => $formView,
    'datagrid' => $datagrid,
    'csrf_token' => $this->getCsrfToken('sonata.browser'),
    'root_category' => $rootCategory,
    'formats' => $formats,
    'export_formats' => [],
]);
```

I am not sure if 'sonata.browser' should be used or another name.
I can make a PR with this if needed 👍

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is broken on next major.

Closes #697

### CODE REVIEW

File: tests/App/Resources/config/config.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ twig:
     exception_controller: null
     paths:
         - "%kernel.project_dir%/Resources/views"
-    strict_variables: "%kernel.debug%"
+    strict_variables: true
```
Comment (jordisala1991) [inline tests/App/Resources/config/config.yaml:44]: we should probably test with this always enabled. cc @VincentLanglet

**Comment by jordisala1991** [comment]
@antoinemineau can you test with this PR?.

**Comment by antoinemineau** [comment]
@jordisala1991 I tested the PR, it fixes the csrf_token.
All good 👍

File: tests/App/Resources/config/config.yaml (hunk 1 above)
Comment (VincentLanglet) [reply tests/App/Resources/config/config.yaml:44]: I strongly agree. This should be done on every sonata project.
(I just did the same in test and even in prod in my work project).

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**